src/A1_csv_to_json.py

```diff
@@ -1,7 +1,7 @@
 """
 How to use:
-all default input csv"s are stored in "/mvs_eland/inputs/elements/default_csv"
-or the input directory/default_csv.
+all default input csv files are stored in "/mvs_eland/inputs/elements/csv"
+or the input directory/csv.
```

Comment:
```suggestion
or their path is provided by the input directory.
```

I changed the implicit existence of the `csv` folder within the input path in the code